selvpc/networking_test.go

```diff
@@ -44,26 +44,42 @@ func TestGetIPVersionFromPrefixLength(t *testing.T) {
 func TestSubnetsMapsFromStructs(t *testing.T) {
 	subnetsStructs := []subnets.Subnet{
```

Comment:
It would be great if you added subnets examples without all fields.
Not all fields will be populated in case of reading a VRRP subnet so we want to make sure that we're getting an expected map.